torch/testing/_internal/common_methods_invocations.py

```diff
@@ -863,6 +863,17 @@ def reference_sigmoid(x):
                                     dtypes=[torch.bfloat16]),),
                        assert_autodiffed=True,
                        promotes_integers_to_float=True),
+        UnaryUfuncInfo('digamma',
+                       ref=scipy.special.digamma,
+                       decorators=(precisionOverride({torch.float16: 5e-1}),),
+                       skips=(SkipInfo('TestUnaryUfuncs', 'test_reference_numerics',
```

Comment:
What's going on with this failure? If the failure is real we should file an issue and cite it

Reply:
It think this was stray. Complex Dtype is not supported.
Removed!
Thanks!
https://github.com/pytorch/pytorch/blob/b82855dddfa2a1cf8d6effab15d0df87b5848569/torch/testing/_internal/common_methods_invocations.py#L873